src/oop_ext/foundation/callback/_tests/test_callback.py

```diff
@@ -687,18 +687,18 @@ def BeforeMethod(*args):
         Before(c.Method, BeforeMethod)
         After(c.Method, AfterMethod)
 
-        mocked = mocker.patch(
-            "oop_ext.foundation.callback.HandleErrorOnCallback", autospec=True
-        )
-        assert c.Method(10) == 20
+        # Now behavior changed and it will fail on first callback error
+        with pytest.raises(RuntimeError):
+            assert c.Method(10) == 20
+
         assert self.before_called == 1
-        assert self.after_called == 1
-        assert mocked.call_count == 2
+        assert self.after_called == 0
+
+        with pytest.raises(RuntimeError):
```

Comment:
This second call and check for raised error is not nescessary.

Reply:
Removed